cmd/all-in-one/main.go

```diff
@@ -307,12 +308,23 @@ func initTracer(metricsFactory metrics.Factory, logger *zap.Logger) io.Closer {
 	return closer
 }
 
-func createMetricsQueryService(factory *metricsPlugin.Factory, v *viper.Viper, logger *zap.Logger) (querysvc.MetricsQueryService, error) {
-	if err := factory.Initialize(logger); err != nil {
+func createMetricsQueryService(
+	metricsReaderFactory *metricsPlugin.Factory,
+	metricsReaderMetricsFactory metrics.Factory,
```

Comment:
Group logger and metrics next to each other

Reply:
Addressed in [a41b9be](https://github.com/jaegertracing/jaeger/pull/3667/commits/a41b9bebc8fd6932cadd56c86fb9d97229c591f3).